README.md

```diff
@@ -1,3 +1,273 @@
+|`domain`| Unique name| Basis domain for DNS entries. As a best practice, use an individual prefix together with the cluster domain of your base cluster.|`vedge.gcp.dev.k8s.jacksgrocerystore.com`|
+|`iaas`| Fixed value | IaaS provider you would like to install Gardener on. | `gcp` |
+|`region`|IaaS provider specific| Region where your Kubernetes base cluster is deployed. | `europe-west1` (GCP)|
+
+### landscape.networks
+```yaml
+networks:                                 # CIDR IP ranges of base cluster
+  nodes: <CIDR IP range>                                  
+  pods: <CIDR IP range>                                   
+  services: <CIDR IP range>
+```
+CIDR ranges of your base cluster.
+
+Finding out CIDR ranges of your cluster is not trivial. For example, GKE only tells you a "pod address range" which is actually a combination of pod and service CIDR. However, since the `kubernetes` service typically has the first IP of the service IP range and most methods to get a kubernetes cluster tell you at least something about the CIDRs, it is usually possible to find out the CIDRs with a little bit of educated guessing.
+
+### landscape.iaas
+```yaml
+iaas:
+  region: <major region>-<minor region>             # region that Gardener will use for seed clusters
+  zones:
+    - <major region>-<minor region>-<zone>          # Example: europe-west1-b
+    - <major region>-<minor region>-<zone>          # Example: europe-west1-c     
+    - <major region>-<minor region>-<zone>          # Example: europe-west1-d
+  credentials:                                      # credentials to get access to the seed cluster through service account
+```
+Contains the information where Gardener will create seed clusters. By default, the *initial* seed component will create a seed resource using your base cluster as seed cluster.
+
+| Field | Type | Description | Example |Iaas Provider Documentation |
+|:------|:--------|:--------|:--------|:--------|
+|`region`|IaaS provider specific|Region where Gardener will create seed clusters and shoot clusters. | `europe-west1`|[GCP (HowTo)](https://cloud.google.com/kubernetes-engine/docs/how-to/managing-clusters#viewing_your_clusters), [GCP (overview)](https://cloud.google.com/docs/geography-and-regions); [AWS (HowTo)](https://docs.aws.amazon.com/cli/latest/reference/eks/describe-cluster.html), [AWS (Overview)](https://docs.aws.amazon.com/general/latest/gr/rande.html)|
+|`zones`|IaaS provider specific|Zones where Gardener will create seed clusters and shoot clusters. |`europe-west1-b`|[GCP (HowTo)](https://cloud.google.com/kubernetes-engine/docs/how-to/managing-clusters#viewing_your_clusters), [GCP (overview)](https://cloud.google.com/docs/geography-and-regions); [AWS (HowTo)](https://docs.aws.amazon.com/cli/latest/reference/eks/describe-cluster.html), [AWS (Overview)](https://docs.aws.amazon.com/general/latest/gr/rande.html)|
```

Comment:
For Azure: remove `zones` completely.

Reply:
note: remove complete zones: block for Azure

Reply:
done